osh/builtin_comp.py

```diff
@@ -406,7 +433,10 @@ def Run(self, cmd_val):
 
     if arg.s:
       if cur.startswith('--') and '=' in cur:  # Split into flag name and value
-        prev, cur = cur.split('=', 1)
+        # mycpp: rewrite of multiple-assignment
+        scur = cur.split('=', 1)
```

Comment:
please use mylib.split_once(cur, '=') here

This will avoid a bit of allocation and is shorter :)